README.md

```diff
@@ -49,11 +49,67 @@ If a function should never be called from another contract, it should be marked
 
 #### 1. Errors
 
-##### A. Prefer custom errors.
+##### A. Using Custom Errors Over Require
 
-Custom errors are in some cases more gas efficient and allow passing useful information.
+Utilize custom errors instead of `require` statements for clearer and more gas-efficient error handling. Solidity 0.8.26 supports the use of custom errors with `require`.
 
-##### B. Custom error names should be CapWords style.
+```solidity
+error InsufficientFunds(uint256 requested, uint256 available);
+
+function withdraw(uint256 amount) public {
+    if (amount > balance) {
+        revert InsufficientFunds(amount, balance);
+    }
+    balance -= amount;
+}
+```
+
+> [!TIP]
```

Comment:
I'm hesitant to introduce this format given we don't use it elsewhere

Reply:
I believe the new admonitions on Github provide significant value and improve visibility.  I have incorporated them into my [Solidity style guide](https://github.com/Aboudjem/solidity-style-guide)